tasks/terraform-smoke.yml

```
@@ -13,6 +13,7 @@ params:
   WORKSPACE: default
   USE_HTTPS: 'true'
   TF_VAR_USE_CONTAINERD: "false"
```

Not that it really matters but

```suggestion
  TF_VAR_ENABLE_CONTAINERD: "false"
```

---

Or just remove it? The default is already set in terraform